flutter/android/android-docker.mk

```diff
@@ -16,10 +16,18 @@
 DOCKER_IMAGE_TAG?=mlcommons/mlperf_mobile_flutter
 
 flutter_docker_postfix=$(shell id -u)
+
+ifeq (${WITH_INTERNAL_DOCKER_OPTION},1)
+host_ip=$(shell nslookup docker-registry.qualcomm.com | grep -n Address | grep ^8 | cut -c12-)
+internal_docker_option=--add-host=docker:${host_ip}
+else
+internal_docker_option=
+endif
```

Comment:
Please move this part to a Makefile inside the `mobile_back_qti` folder, since it's only for Qualcomm.

Reply:
removed these.